src/wallet/rpcdump.cpp

```diff
@@ -1160,3 +1160,66 @@ UniValue importmulti(const JSONRPCRequest& mainRequest)
 
     return response;
 }
+
+UniValue rescanblockchain(const JSONRPCRequest& request)
+{
+    CWallet * const pwallet = GetWalletForJSONRPCRequest(request);
+    if (!EnsureWalletIsAvailable(pwallet, request.fHelp)) {
+        return NullUniValue;
+    }
+
+    if (request.fHelp || request.params.size() > 2) {
+        throw std::runtime_error(
+            "rescanblockchain (\"startheight\") (\"stopheight\")\n"
```

Comment:
What I mean is that with `bitcoin-cli rescanblockchain 10` `10` will always be `startheight` and to specify only `stopheight` you have to call `bitcoin-cli rescanblockchain null 10`, hence `stopheight` depends on `startheight`.